app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/OverviewFragment.kt

```diff
@@ -13,13 +13,12 @@ import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModel
 import edu.uc.kovaciad.slicetracker.R
 
-class OverviewFragment : Fragment() {
+class OverviewFragment : SuperFragment() {
```

Comment:
Changing subFragments to extend the newly created super fragment as shown in this class material video here: https://www.youtube.com/watch?v=pwZQRw08mP0